src/views/imgur.ejs

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Image Upload</title>
+  </head>
+  <body>
+    <h1>Image Upload</h1>
+    <form
+      action="/api/v1/user/<%= userId %>/uploadProfile"
```

Comment:
Sorry, actually looking at `expressServer.js` closer I see this should be "users" plural:

```suggestion
      action="/api/v1/users/<%= userId %>/uploadProfile"
```

Reply:
I have noticed as well in previous comment, corrected commit 2127735a8baeed895a8a5f80939c8e241567cf1f